### ISSUE TITLE
HTMLResource description incorrectly specifies URI requirement

### ISSUE THREAD

**Issue by wgrose** [InteractiveAdvertisingBureau/vast#22]
## HTMLResource description specifies that it should contain a URI instead of an html snippet/document.

![htmlresource_iframeresource](https://user-images.githubusercontent.com/20191/53044566-4152af80-3440-11e9-93d8-ec9f27818d6f.png)

## Examples

My understanding is that HTMLResource should look something like:

```
<HTMLResource><![CDATA[<html><body>I'm a companion</body></html>]]></HTMLResource>
```

Where as the IFrameResource would be a remote resource:

```
<IFrameResource><![CDATA[http://adserver.com/htmlresourcefile.htm]]></IFrameResource>
```

## Also
There are a few places that this would need to change, such as section 3.15.

I see this bug:
https://github.com/InteractiveAdvertisingBureau/vast/issues/6

Which also mentions HTMLResource and a HTMLResourceType but I couldn't find the latter mentioned in the current spec.

**Comment by richardrapp** [comment]
It looks like a copy-paste error, and there are multiple errors.

The description under TOC and Schema for all three of the Resources (StaticResource, IframeResource, HTMLResourcce) say "The URI to a static creative file to be used".

In the content section however, Iframe's looks correct, but HTMLResource's is the static one again.

It appears StaticResource was copied for both, and the content section for IframeResource was the only part of the copy that was updated.

**Comment by alexchugunov** [comment]
We noticed that too at Pandora. Looks like a copy&paste error, agreed.

**Comment by amitshetty** [comment]
I was just looking at this - I dont see the edits in VAST 4.2. Did I miss them somehow?

**Comment by richardrapp** [comment]
Which edits do you mean? Vast 4.2 looks the same to me as what's cited above.

**Comment by amitshetty** [comment]
Thats the point. We had a discussion in the WG to fix the above, and I see a comment in the doc that @wgrose was going to suggest some edits to fix the above. I could have sworn he did make the edits, but I don't see it in any of the docs I have.

**Comment by wgrose** [comment]
Sorry, I kept meaning to get to it but never did.

**Comment by amitshetty** [comment]
@wgrose - I am updating the doc right now. Please email me at amit @ iabtechlab.com so I have your email to send (for you to review). 
Thanks!

**Comment by amitshetty** [comment]
Fixed in VAST 4.2 corrections update posted recently.
